app/src/main/java/io/github/droidkaigi/confsched/activity/ActivityNavigator.java

```diff
@@ -15,8 +15,8 @@ public void showSessionDetail(@NonNull Activity activity, @NonNull Session sessi
         SessionDetailActivity.startForResult(activity, session, requestCode);
     }
 
-    public void showMain(@NonNull Activity activity) {
-        MainActivity.start(activity);
+    public void showMain(@NonNull Activity activity, boolean shouldRefresh) {
+        MainActivity.start(activity, shouldRefresh);
```

Comment:
After language setting changed, it is necessary to reload sessions.json of new language.